packages/react/src/ArqueroDetailsList/renderers/SmartArrayCell.tsx

```diff
@@ -33,9 +34,12 @@ export const SmartArrayCell: React.FC<RichCellProps> = memo(
 				<Case condition={values.length <= cellWidth}>
 					<SparkbarCell {...props} />
 				</Case>
-				<Case condition={values.length > cellWidth}>
+				<Case condition={values.length > cellWidth && !arrayAsDropdown}>
```

Comment:
I think if we are going to have `arrayAsDropdown` as a separate feature rather than automatic if `smartCells` is turned on, then we should probably override _any_ array cell to be a dropdown instead of a chart. In other words, either you get a plan text array cell, a smart cell with charts, or a dropdown.